plugins/presentation/src/components/internal/presenters/value/StringEditor.svelte

```diff
@@ -21,14 +21,24 @@
   export let attribute: AttrModel
   export let maxWidth: number = 300
   export let editable: boolean
+  export let textWrap: boolean
```

Comment:
I do not think we should pass textWrap here, presenter should decide how to show data, it should not be controlled from outside, maxWidth is enough to decide I think.